public/locales/en/panel-scene.json

```diff
@@ -0,0 +1,118 @@
+      "this-is-irreversible": "This action is irreversible and will also remove all nodes in the scene tree that depend on this node!"
+    },
+    "pop-out": "Pop out"
+  },
+  "scene-graph-node-view": {
+    "not-found-info": "This scene graph node does not exist. It might have been removed since this view was opened.",
+    "renderable": {
+      "title": "Renderable",
+      "tooltip": "Properties that control the visuals of this object",
+      "no-renderable": "This scene graph node has no renderable"
+    },
+    "transform": {
+      "title": "Transform",
+      "tooltip": "Properties that control the position, scale, and orientation of this object",
+      "no-transform": "This scene graph node has no transform"
+    },
+    "timeframe": {
+      "title": "Time",
+      "tooltip": "Information about the time frame for when the object is visible",
+      "status": {
+        "label": "Current status",
+        "tooltip-active": "This object is currently active and will be visible",
+        "tooltip-inactive": "This object is currently inactive due to its time frame and will not be visible",
+        "active": "Active",
+        "inactive": "Inactive"
+      },
+      "info": "The object will not be visible outside the specified time range."
+    },
+    "info": {
+      "title": "Info",
+      "tooltip": "Information about the scene graph node and its asset"
+    },
+    "other": {
+      "title": "Other",
+      "tooltip": "Other properties of this scene graph node"
+    }
+  },
+  "scene-tree": {
+    "featured-scene-tree": {
+      "current-focus-label": "Current Focus",
+      "current-aim-label": "Current Aim",
+      "quick-access-label": "Quick Access"
+    },
+    "scene-tree":{
+      "filter-list-placeholder": "Search for a node...",
+      "search-keys": {
+        "name": "Name",
+        "guiPath": "GUI Path"
+      },
+      "collapse-all-tooltip": "Collapse all",
+      "collapse-all-aria-label": "Collapse all nodes",
+      "expand-all-tooltip": "Expand all",
+      "expand-all-aria-label": "Expand all nodes"
+    },
+    "scene-tree-filters": {
+      "reset-filters": "Reset filters",
+      "settings-tooltip": "Additional settings to filter search results",
+      "settings-aria-label": "Open search filter settings",
+      "menu-label-filters": "Filters",
+      "show-only-visible": {
+        "label": "Show only visible",
+        "info": "Show only nodes that are currenlty visible, meaning enabled and not faded out."
+      },
+      "show-only-focusable": {
+        "label": "Show only focusable",
+        "info": "Hide scene graph nodes that are not markes as focusable, meaning that they cannot be directly set as the focus node in the scene."
+      },
+      "show-gui-hidden": {
+        "label": "Show objects with GUI hidden flag",
+        "info": "Show scene graph nodes that are marked as hidden in the GUI part ofthe asset. These are otherwise hidden in the interface."
```

Comment:
```suggestion
        "info": "Show scene graph nodes that are marked as hidden in the GUI part of the asset. These are otherwise hidden in the interface."
```